docs/topics/device_connection.md

```diff
@@ -0,0 +1,265 @@
+platform. This interaction with the Platform can be implemented with whichever programming language your device supports.
+No SDKs are provided for south bound interactions.
+
+In this case, a typical measure report from our Weather Station will consist on a request like the following
+one:
+
+    POST /iot/d?k=801230BJKL23Y9090DSFL123HJK09H324HV8732&i=ws1956672
+    Content-Type: text/plain
+
+    t|22|h|78
+
+You should notice several things in this request:
+
+* First of all, the path is '/iot/d'. The notification path will depend on the protocol used. This path is the one assigned
+to the Ultralight 2.0 protocol
+
+* Also, the query parameters indicate the API Key defined in the first section, and the device ID defined in the previous
+one.
+
+* The payload of the request contains the measures, sepparated by bars. The names of the attributes used to report the
+measure are the ones defined in the `object_id` attributes of the device provisioning.
+
+* The `Content-Type` header is mandatory and **must** have the value `text/plain`.
+
+If everything is OK, the system will answer with a 200 OK code, and an empty answer.
+
+This measure will give rise to a Context Entity like the following one in the Context Broker:
+
+      {
+         "type": "Device",
+         "isPattern": "false",
+         "id": "WeatherStation1",
+         "attributes": [
+           {
+             "name": "temperature",
+             "type": "degrees",
+             "value": "22"
+           },
+           {
+             "name": "humidity",
+             "type": "percentage",
+             "value": "78"
+           },
+           {
+             "name": "location",
+             "type": "geo:point",
+             "value": "22.1, -17"
+           },
+           {
+             "name": "reference",
+             "type": "string",
+             "value": "917834508965243"
+           }
+         ]
+      }
+
+### Sending a command
+
+The Platform gives also the opportunity for the devices to receive commands or information from the platform. To illustrate
+this point, we will show how to send some configuration information to a device, using polling commands, i.e.: commands
+that do not require the device to be online to be receieved, but that are retrieved when the device connects to the platform.
+
+Commands are also defined in the device provisioning. It can be seen in the `commands` section of the provisioning payload
+in the provisioning section above.
+
+Information for the command execution is sent using NGSI queries to the Context Broker, as if we were going to update
+directly the Context Entity that represents the device. As an example, we will send the `openUmbrella=false` configuration
+order with the following request:
+
+    PUT /v2/entities
+    Content-Type: application/json
+
+    {
+      "id": "WeatherStation1",
+      "type": "Device",
+      "configuration": {
+        "type" : "command",
+        "value" : "openUmbrella=false"
```

Comment:
`=` is a forbidden char in the API... This example shouldn't work (if it actually works it would be a bug at CB